internal/cephfs/core/metadata.go

```diff
@@ -35,6 +35,7 @@ var ErrSubVolMetadataNotSupported = errors.New("subvolume metadata operations ar
 func (s *subVolumeClient) supportsSubVolMetadata() bool {
 	if _, keyPresent := clusterAdditionalInfo[s.clusterID]; !keyPresent {
 		clusterAdditionalInfo[s.clusterID] = &localClusterState{}
+		clusterAdditionalInfo[s.clusterID].subVolumeGroupsCreated = make(map[string]bool)
```

Comment:
this is copy/pasted in multiple places now, that is really bad. Please create a `newLocalClusterState()` function that initializes the object and use that everywhere.